.eleventy.js

```diff
@@ -262,6 +269,7 @@ module.exports = function(eleventyConfig) {
 	eleventyConfig.addNunjucksAsyncShortcode("image", shortcodes.image);
 	eleventyConfig.addShortcode("avatarlocalcache", shortcodes.avatar);
 	eleventyConfig.addShortcode("communityavatar", shortcodes.communityAvatar);
+	eleventyConfig.addShortcode("opencollectavatar", shortcodes.getOpenCollectiveAvatarHtml);
```

Comment:
Nit: do you want “opencollectavatar” or “opencollectiveavatar”? (collect vs collective)